docs/csv-description.md

```diff
@@ -19,6 +19,25 @@ The Koku Metrics Operator (`koku-metrics-operator`) collects the metrics require
 * PersistentVolumeClaim (PVC) configuration: The KokuMetricsConfig CR can accept a PVC definition and the operator will create and mount the PVC. If one is not provided, a default PVC will be created.
 * Restricted network installation: this operator can function on a restricted network. In this mode, the operator stores the packaged reports for manual retrieval.
 
+## New in v3.3.2:
+* Leader election timing is now configurable via the environment variables below. To update these settings, [modify the variables in a Subscription](https://github.com/operator-framework/operator-lifecycle-manager/blob/5a01f50258003e248bd5630df0837fe0bb0f1cb7/doc/design/subscription-config.md) and apply the changes.
+  
+  ```
+      kind: Subscription
+      metadata:
+      ...
+      spec:
+        ...
+        config:
+          env:
+            - name: LEADER_ELECTION_LEASE_DURATION  # sets the lease timeout duration
+              value: "60s"
+            - name: LEADER_ELECTION_RENEW_DEADLINE  # specifies the deadline to renew the lease
+              value: "40s"
+            - name: LEADER_ELECTION_RETRY_PERIOD    # defines the interval between retry attempts
+              value: "15s"
```

Comment:
I think the variable names are obvious, so the comment doesn't provide much value. It would be good to mention that the value needs to be a duration. What format is the variable expected to be?

Reply:
updated to include expected format for values